README.md

```diff
@@ -7,6 +7,27 @@ services used by Mozilla Hubs for local development.[^1]
 security or scalability.  Additionally the permissions files were generated for
 development purposes only.
 
+## Windows Prerequisites
+### Summary
+- [Install WSL2](https://learn.microsoft.com/en-us/windows/wsl/install)
+- Ensure Git checks out Unix line endings
+- Use Git Bash to run scripts
+
+### Install WSL2
+Docker Desktop runs more quickly when it can use a WSL2-based engine. There are also other benefits to installing WSL2, including being able to run scripts and programs that your Unix friends share with you.
```

Comment:
We might want to note that this is an optional dependency.

Reply:
Total tangent, but do folks still use Cygwin? I remember that being a boon to cross-platform devs working on Windows.

Reply:
I stopped using Cygwin in exchange for Git Bash/MINGW64 several years ago. I remember Cygwin blowing my mind when I first started using it. "I can do _what_ on the command line now??"

Reply:
(Comment addressed!)